ruth/data/map.py

```diff
@@ -117,15 +117,18 @@ def k_fastest_paths(self, origin, dest, k):
         except NetworkXNoPath:
             return None
 
+    def to_hdf5(self):
+        return save_graph_to_hdf5(self.simple_network, 'data/map.hdf5')
```

Comment:
Please store the filepath somewhere, e.g. as a field of `Map`. We will need to access it when sending the path to the distributed nodes.